<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QWidget" name="SettingsDialog">
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/tspsg.qrc">
    <normaloff>:/images/icons/preferences_system.png</normaloff>:/images/icons/preferences_system.png</iconset>
  </property>
  <widget class="Line" name="lineHorizontal">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>2</height>
    </size>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="bgGrey" native="true">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>240</red>
         <green>240</green>
         <blue>240</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>240</red>
         <green>240</green>
         <blue>240</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>240</red>
         <green>240</green>
         <blue>240</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <widget class="QPushButton" name="buttonCancel">
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="statusTip">
     <string>Close without saving preferences</string>
    </property>
    <property name="text">
     <string>&amp;Cancel</string>
    </property>
    <property name="icon">
     <iconset resource="../resources/tspsg.qrc">
      <normaloff>:/images/icons/button_cancel.png</normaloff>:/images/icons/button_cancel.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonOK">
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="statusTip">
     <string>Save new preferences</string>
    </property>
    <property name="text">
     <string>&amp;OK</string>
    </property>
    <property name="icon">
     <iconset resource="../resources/tspsg.qrc">
      <normaloff>:/images/icons/button_ok.png</normaloff>:/images/icons/button_ok.png</iconset>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="buttonHelp">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>Help</string>
    </property>
    <property name="statusTip">
     <string>Get preferences help</string>
    </property>
    <property name="text">
     <string>Help</string>
    </property>
    <property name="icon">
     <iconset resource="../resources/tspsg.qrc">
      <normaloff>:/images/icons/help_hint.png</normaloff>:/images/icons/help_hint.png</iconset>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="bgWhite" native="true">
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <widget class="QGroupBox" name="groupRandomSettings">
    <property name="title">
     <string>Random values:</string>
    </property>
    <layout class="QVBoxLayout">
     <item>
      <layout class="QHBoxLayout" name="_2">
       <item>
        <widget class="QLabel" name="labelRandMin">
         <property name="text">
          <string>&amp;Minimal:</string>
         </property>
         <property name="buddy">
          <cstring>spinRandMin</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinRandMin">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="statusTip">
          <string>Minimum for random number generation</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="_3">
       <item>
        <widget class="QLabel" name="labelRandMax">
         <property name="text">
          <string>Ma&amp;ximal:</string>
         </property>
         <property name="buddy">
          <cstring>spinRandMax</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinRandMax">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="statusTip">
          <string>Maximum for random number generation</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupOutputSettings">
    <property name="title">
     <string>Output settings:</string>
    </property>
    <widget class="QPushButton" name="buttonFont">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="statusTip">
      <string>Font face for printing</string>
     </property>
     <property name="text">
      <string>&amp;Font...</string>
     </property>
     <property name="icon">
      <iconset resource="../resources/tspsg.qrc">
       <normaloff>:/images/icons/fonts.png</normaloff>:/images/icons/fonts.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonColor">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="statusTip">
      <string>Font color for printing</string>
     </property>
     <property name="text">
      <string>&amp;Color...</string>
     </property>
     <property name="icon">
      <iconset resource="../resources/tspsg.qrc">
       <normaloff>:/images/icons/color_line.png</normaloff>:/images/icons/color_line.png</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="cbAutosize">
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="statusTip">
     <string>Automatically resize rows and columns to their contents</string>
    </property>
    <property name="text">
     <string>Autosize of rows and columns</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../resources/tspsg.qrc"/>
 </resources>
 <connections/>
</ui>
